tests/integration/golang/auth/service_with_user_auth_from_config_test.go

```diff
@@ -0,0 +1,432 @@
+package auth
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+	"github.com/zeebo/assert"
+	"gopkg.in/yaml.v3"
+
+	aimResponse "github.com/G-Research/fasttrackml/pkg/api/aim/response"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow"
+	mlflowResponse "github.com/G-Research/fasttrackml/pkg/api/mlflow/api/response"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/config"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/config/auth"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
+	"github.com/G-Research/fasttrackml/pkg/common"
+	"github.com/G-Research/fasttrackml/pkg/common/api"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type ServiceWithUserAuthFromConfigTestSuite struct {
```

Comment:
New name is a little confusing to me because `Service` prefix -- since none of our `services` are involved. Is this testing the the application/server? or perhaps the application middleware?